src/frontend/src/lib/components/send/SendInputAmount.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+	import { Input } from '@dfinity/gix-components';
+	import { debounce, nonNullish } from '@dfinity/utils';
+	import { parseToken } from '$lib/utils/parse.utils';
+	import { slide } from 'svelte/transition';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { BigNumber } from '@ethersproject/bignumber';
+	import { invalidAmount } from '$lib/utils/input.utils';
+
+	export let amount: number | undefined = undefined;
+	export let tokenDecimals: number | undefined = undefined;
+	export let label: string = $i18n.core.text.amount;
+	export let placeholder: string = $i18n.core.text.amount;
+	export let customValidations: ((value: BigNumber) => Error | undefined)[] = [];
+	export let reactivityVariables: (unknown | undefined)[] = [];
+	export let debounceTime: number | undefined = 300;
+
+	let error: Error | undefined;
+
+	const runValidations = (value: BigNumber) => {
```

Comment:
note: I did not reviewed this function given that I suggest to pass only a single validator